src/pixelator/plot/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""Functions for creating plots that are useful with MPX data.
+
+Copyright (c) 2023 Pixelgen Technologies AB.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from matplotlib import cm
+from typing import Literal
+
+from pixelator.graph import Graph
+from pixelator.marks import experimental
+
+
+def _unit_sphere_surface(m, n):
+    u = np.linspace(0, 2 * np.pi, m)
+    v = np.linspace(0, np.pi, n)
+
+    X = np.outer(np.cos(u), np.sin(v))
+    Y = np.outer(np.sin(u), np.sin(v))
+    Z = np.outer(np.ones(np.size(u)), np.cos(v))
+    return X, Y, Z
```

Comment:
It might improve readability if `m` and `n` params are more descriptive (e.g. `azimuthal_resolution`/`horizontal_resolution` and `polar_resolution`/`vertical_resolution`). 
Same goes for `u` and `v`. I would suggest something explicit like `azimuthal_angles`/`horizontal_angles` and `polar_angles`/`vertical_angles`

Reply:
Good point. I'll update that.